src/store/sync/conversation.ts

```diff
@@ -30,7 +29,10 @@ export const handleModifiedConversationsReducer = (
 ): void => {
 	forEach(payload, (conv) => {
 		if (conv?.id && state?.conversations?.[conv.id]) {
-			state.conversations[conv.id] = merge(state.conversations[conv.id], conv);
+			state.conversations[conv.id] = {
+				...merge(state.conversations[conv.id], conv),
```

Comment:
I am pretty sure this is unnecessary

Reply:
@nubsthead Here is the proof,

Lets suppose I have a conversation which have 2 tags 'one' & 'two'
`let c1={
               id:1,
             tags:['one','two']
};`
later if you remove the tag 'two' from the conversation, you will receive
`{
![Screenshot 2022-04-22 at 11 05 54 AM](https://user-images.githubusercontent.com/74703491/164610079-8511a603-f05e-4183-bdbb-caa04726b7dd.png)

               id:1,
             tags:['one']
}`
and if you merge them you'll get the output as 
`{
               id:1,
             tags:['one','two']
};
` which is incorrect